Formula/stanford-ner.rb

```diff
@@ -0,0 +1,20 @@
+class StanfordNer < Formula
+  desc "Stanford NLP Group's implementation of a Named Entity Recognizer."
+  homepage "http://nlp.stanford.edu/software/CRF-NER.shtml"
+  url "http://nlp.stanford.edu/software/stanford-ner-2015-04-20.zip"
+  version "3.5.2"
```

Comment:
If you visit the URL listed as the [homepage] and scroll all the way to the bottom, you can find a list of revisions.

The most recent version is 3.6.0. This formula packages the 3.5.2 version, because it's the most recent version known to work with the popular NLTK Python package, and I figured it's easier to upgrade a package later than to downgrade a package later.

[homebrew]: http://nlp.stanford.edu/software/CRF-NER.shtml